patches/nan/api_remove_allcan_read_write.patch

```diff
@@ -0,0 +1,87 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: deepak1556 <hop2deep@gmail.com>
+Date: Fri, 10 Nov 2023 22:51:53 +0900
+Subject: Remove AllCan Read/Write
+
+Refs https://chromium-review.googlesource.com/c/v8/v8/+/5006387
+Should be upstreamed.
+
+Module version checks are made against 119 since that is the current assigned version
+for Electron 28 in https://github.com/nodejs/node/blob/main/doc/abi_version_registry.json.
+The version should be updateed to the one assinged for Electron 29 when it is available.
+
+Steps for upstreaming this patch:
+
+- Get a new module version assigned for Electron 29 in nodejs/node
+- Update NODE_21_0_MODULE_VERSION to the new version number
+- Upstream patch to nodejs/nan before Electron 29 is branched
```

Comment:
I'll add these to the Release Board Stable items for Electron 29, just so we don't lose track of them 👍